pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
+      {% endif %}
+    }
+
+    /* TODO: Add hover color to model or dynamically generate */
+    #primary-cta:hover {
+      opacity: 0.9;
+      transition: 0.25s;
+    }
+
+    #primary_content {
+      background-size: cover;
+      {% if page.primary_content_background_color %}
+        background-color: #{{ page.primary_content_background_color }};
+      {% endif %}
+
+      {% if page.primary_content_text_color %}
+        color: #{{ page.primary_content_text_color }};
+      {% endif %}
+
+      {% if page.primary_content_background_image %}
+        {% image page.primary_content_background_image original as primary_img %}
+        background-image: url({{ primary_img.url }});
+      {% endif %}
+    }
+
+    #primary_content a {
+      {% if page.primary_content_text_color %}
+        color: #{{ page.primary_content_text_color }};
+      {% endif %}
+    }
+
+    #secondary_content {
+      background-size: cover;
+      {% if page.secondary_content_background_color %}
+        background-color: #{{ page.secondary_content_background_color }};
+      {% endif %}
+
+      {% if page.secondary_content_text_color %}
+        color: #{{ page.secondary_content_text_color }};
+      {% endif %}
+
+      {% if page.secondary_content_background_image %}
+        {% image page.secondary_content_background_image original as secondary_img %}
+        background-image: url({{ secondary_img.url }});
+      {% endif %}
+    }
+
+    #secondary_content a {
+      {% if page.secondary_content_text_color %}
+        color: #{{ page.secondary_content_text_color }};
+      {% endif %}
+    }
+
+    #footer, .microsite .footer {
+      background-size: cover;
+      {% if page.custom_footer_background_color %}
+        background-color: #{{ page.custom_footer_background_color }};
+      {% endif %}
+
+      {% if page.custom_footer_text_color %}
+        color: #{{ page.custom_footer_text_color }};
+      {% endif %}
+
+      {% if page.custom_footer_background_image %}
+        {% image page.custom_footer_background_image original as secondary_img %}
+        background-image: url({{ secondary_img.url }});
+      {% endif %}
+    }
+
+    #footer a, .microsite .footer a {
+      {% if page.custom_footer_text_color %}
+        color: #{{ page.custom_footer_text_color }};
+      {% endif %}
+    }
+
+    .microsite-content .can_button, .microsite-content #can_embed_form input[type="submit"], .microsite-content #can_embed_form .button, .microsite-content #donate_auto_modal input[type="submit"], .microsite-content #donate_auto_modal .button {
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}} !important;
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}} !important;
+      {% endif %}
+    }
+  </style>
+{% endblock %}
+
+<div class="microsite">
+
+{% block notification_banner %}
+<!-- TODO: conditional notification banner on page model -->
```

Comment:
I thought this would follow standard header conditional logic/toggle for now. But maybe this is better? Do we not want to ever show notification banner on a microsite page? Not sure.

Reply:
There are use cases where we would want both nav and custom header I think.

Reply:
Do you mean notification banner instead?

Reply:
Ha I did not but I realize I misread this. I see them as two distinct elements but perhaps they shouldn't be.

Reply:
Ok I think it's fine to leave as is for now. At some point we can add in conditional toggle for notification banner if we need it.